### PR TITLE
www: redirects for renamed openjsf content

### PR DESCRIPTION
**Pull request by rvagg**
Fixes: https://github.com/nodejs/build/issues/2194

would appreciate nginx config sanity checking before I try putting this on the server.

### CODE REVIEW

File: ansible/www-standalone/resources/config/nodejs.org [hunk 1]
```diff
@@ -179,7 +179,13 @@ server {
     listen [::]:443 ssl http2;
     server_name foundation.nodejs.org;
 
-    return 301 https://openjsf.org/;
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Nasa_FNL.pdf$         https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-NASA.pdf permanent;
```
Comment (brianwarner) [inline ansible/www-standalone/resources/config/nodejs.org]: Looks like the redirect is maybe still pointing to foundation.nodejs.org?  I'll add some suggestions which point it to the direct location on openjsf.org instead.

```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Nasa_FNL.pdf$         https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-NASA.pdf permanent;
```

File: ansible/www-standalone/resources/config/nodejs.org [hunk 2]
```diff
@@ -179,7 +179,13 @@ server {
...
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Fusion_Final.pdf$     https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-Fusion.pdf permanent;
```
Comment (brianwarner) [inline ansible/www-standalone/resources/config/nodejs.org]:
```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Fusion_Final.pdf$     https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-Fusion.pdf permanent;
```

File: ansible/www-standalone/resources/config/nodejs.org [hunk 3]
```diff
@@ -179,7 +179,13 @@ server {
...
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Walmart_final-1.pdf$  https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-Walmart.pdf permanent;
```
Comment (brianwarner) [inline ansible/www-standalone/resources/config/nodejs.org]:
```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_Walmart_final-1.pdf$  https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-Walmart.pdf permanent;
```

File: ansible/www-standalone/resources/config/nodejs.org [hunk 4]
```diff
@@ -179,7 +179,13 @@ server {
...
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_HomeAway.pdf$         https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-HomeAway.pdf permanent;
```
Comment (brianwarner) [inline ansible/www-standalone/resources/config/nodejs.org]:
```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CaseStudy_HomeAway.pdf$         https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-HomeAway.pdf permanent;
```

File: ansible/www-standalone/resources/config/nodejs.org [hunk 5]
```diff
@@ -179,7 +179,13 @@ server {
...
+    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CapitalOne_FINAL_casestudy.pdf$ https://foundation.nodejs.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-CapitalOne.pdf permanent;
```
Comment (brianwarner) [inline ansible/www-standalone/resources/config/nodejs.org]:
```suggestion
    rewrite ^/wp-content/uploads/sites/50/2017/09/Node_CapitalOne_FINAL_casestudy.pdf$ https://openjsf.org/wp-content/uploads/sites/84/2020/02/Case_Study-Node.js-CapitalOne.pdf permanent;
```

**Comment by rvagg** [comment]
good thing I got you to check! whoops.
deployed

**Pull request merged**